ip.go

```diff
@@ -51,22 +51,21 @@ func IsThinWaist(m ma.Multiaddr) bool {
 	}
 }
 
+var localPrefixes = [][]byte{
+	{ma.P_IP4, 127}, // 127.*
```

Comment:
I'd prefer to have this out in a const/var with a comment

Reply:
It's already a global var (can't be a const, unfortunately) but I've moved it up, documented it, and renamed it to something more accurate.